### PR TITLE
Break east API in favor of consistency and logic

### PR DESCRIPTION
**Pull request by MarkoSagadin**
# Description

See the commit messages.

## Areas of interest

Created changes.

## Checklist

- [x] Source code documentation for all newly added or changed functions was added/updated.
- [x] All customer facing technical documentation was updated.

## After-review steps

* Code author will merge the PR by himself/herself.

### CODE REVIEW

**Comment by MarkoSagadin** [comment]
Current dependencies on/for this PR:
* dev
  * **PR #90** <a href="https://app.graphite.dev/github/pr/IRNAS/irnas-east-software/90" target="_blank">Graphite</a> 
    * **PR #91** <a href="https://app.graphite.dev/github/pr/IRNAS/irnas-east-software/91" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/IRNAS/irnas-east-software/91?utm_source=stack-comment).

File: src/east/system_commands/tooling.py [hunk 1]
```diff
@@ -281,7 +285,7 @@ def _install_codechecker(east: EastContext, exe_path: str):
 [bold green]codechecker install done![/]
 """
 
-packages = [
+supported_tools = [
```
Comment (TjazVracko) [inline src/east/system_commands/tooling.py:290]: Maybe remove conda from the list?

**Review by TjazVracko — APPROVED** [review]

File: src/east/system_commands/tooling.py (hunk 1 above)
Comment (MarkoSagadin) [reply src/east/system_commands/tooling.py:290]: Yes, I will. I don't think that we will need it in the near future.

**Comment by MarkoSagadin** [comment]
### Merge Activity

* **Sep 25, 2:42 AM**: @MarkoSagadin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/IRNAS/irnas-east-software/91).

**Pull request merged**